### PR TITLE
add token dbDEGEN

### PR DESCRIPTION
**Pull request by CaesarDB**
**Description**

To the kind attention of @cfluke-cb @wbnns @taycaldwell

dbDEGEN is the official DEGEN IOU token issued by Deribet.io. The token is redeemable 1-to-1 (1 dbDEGEN = 1 DEGEN) on the platform Deribet.io

Link to Basescan: https://basescan.org/token/0x61431b49d43220a53b4a2583870d69f67ced8d76

### CODE REVIEW

**Review by wbnns — DISMISSED** [review]
@CaesarDB 

Hi there! Could you please add the `"nobridge": true,` option to your `data.json`?

**Comment by CaesarDB** [comment]
@wbnns hi! Just added! Sorry for the miss!

Thanks

File: data/dbDEGEN/data.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "symbol": "dbDEGEN",
+    "decimals": 18,
+    "description": "dbDEGEN is the official DEGEN IOU token issued by Deribet.io. The token is redeemable 1-to-1 (1 dbDEGEN = 1 DEGEN) on the platform Deribet.io",
+    "website": "https://deribet.io/",
+    "twitter": "@deribet_io",
+    "tokens": {
+        "base": {
+            "address": "0x61431b49D43220a53B4a2583870D69F67cEd8d76"
+        },
+        "nobridge": true
```
Comment (wbnns) [inline data/dbDEGEN/data.json:12]: @CaesarDB 

Thanks for the update; this should go up above `tokens`, you can put it under `twitter`. Can you please also fix the spacing and make sure the JSON validates after?

Comment (CaesarDB) [reply data/dbDEGEN/data.json]: @wbnns done - sorry for the back and forth!

**Review by wbnns — APPROVED** [review]
ACK

@CaesarDB Thanks for the updates and for building on Base 🔵

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**